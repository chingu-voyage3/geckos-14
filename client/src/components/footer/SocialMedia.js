import React from 'react';

const SocialMedia = () => {
   return (
       <div className="footer-mediaButtons">
        <p>
        <u>Get in touch: </u>
          <i className="fa fa-facebook-official fa-x2"></i>
          <i className="fa fa-github fa-x2"></i>
          <i className="fa fa-twitter-square fa-x2"></i>
        </p>
        </div>
    );
};

  export default SocialMedia;
