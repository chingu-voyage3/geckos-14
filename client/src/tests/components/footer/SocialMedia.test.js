import React from 'react';
import ReactDOM from 'react-dom';
import SocialMedia from '../../../components/footer/SocialMedia';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<SocialMedia />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
