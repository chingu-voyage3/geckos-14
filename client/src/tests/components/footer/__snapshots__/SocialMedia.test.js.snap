// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="footer-mediabuttons"
>
  <p>
    <u>
      Get in touch:
    </u>
    <i
      className="fab fa-facebook fa-2x"
    />
    <i
      className="fab fa-github fa-2x"
    />
    <i
      className="fab fa-twitter-square fa-2x"
    />
  </p>
</div>
`;
